typescript/cli/ci-test.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+# Optional cleanup for previous runs, useful when running locally
+pkill -f anvil
+docker ps -aq | xargs docker stop | xargs docker rm
+rm -rf /tmp/anvil*
+rm -rf /tmp/relayer
+
+# Setup directories for anvil chains
+for CHAIN in anvil1 anvil2
+do
+    mkdir -p /tmp/$CHAIN /tmp/$CHAIN/state  /tmp/$CHAIN/validator /tmp/relayer
+    chmod -R 777 /tmp/relayer /tmp/$CHAIN
+done
+
+# Optional: remove the --block-time 1 to speedup tests for local runs
```

Comment:
why not remove it here?

Reply:
I noticed some flakiness when it's not there, which is why Asa originally included it I assume. Tbh we may be able to make it work without it but I've spent too much time on this script already 😫